api_server.py

```
@@ -455,36 +466,30 @@ def get_crew_duration_data(id):
 
 def get_appliance_available_data(name):
     now = datetime.now()
```

Ruff (0.15.7)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)